lib/debug/server_cdp.rb

```diff
@@ -489,7 +489,12 @@ def process
                         id: SecureRandom.hex
         when 'Runtime.terminateExecution'
           send_response req
-          exit
+          if SESSION.in_subsession?
+            @q_msg << 'kill!'
+          else
+            @q_msg << 'kill!'
+            pause
+          end
```

Comment:
Feels like it could be simplified a bit?

```suggestion
          @q_msg << 'kill!'
          pause unless SESSION.in_subsession?
```